### DISCUSSION TITLE
Primefaces CSS Style not working

### DISCUSSION THREAD

**Discussion by engegov** [primefaces/community#2843] [PrimeFaces] [answered]
I have a Primefaces 12.0.0 in my application, and need develop a dataView like showcase example, but not rendering css style "product card" and any others.

How can I find out this css code ? 

Thans for advance.

**Comment by melloware** [comment, marked as answer]
Converted this to a discussion not a bug ticket.  

The showcase just uses the built in Saga theme and some custom layout CSS which you can grab off github here:

https://github.com/primefaces/primefaces/blob/master/primefaces-showcase/src/main/webapp/resources/showcase/css/layout/layout.css

**Reply by engegov** [reply]
Thank you so much.
